safe_transaction_service/history/management/commands/setup_service.py

```diff
@@ -75,6 +75,12 @@ def create_task(self) -> Tuple[PeriodicTask, bool]:
         5,
         IntervalSchedule.MINUTES,
     ),
+    CeleryTaskConfiguration(
+        "safe_transaction_service.history.tasks.reindex_master_copies_last_hours",
```

Comment:
Should this be `safe_transaction_service.history.tasks.reindex_last_hours`

Reply:
Sure! Nice catch!

Reply:
Should be fixed now